### ISSUE TITLE
Sync branch 'wip-7-SeparationOfConcerns' with 'main'

### ISSUE THREAD

**Issue by fjbo-net** [cotheca/docs#38]
## Description
Recent changes from the following branches require to update any active branches to avoid data corruption, in terms of methodology and file structure for the project:
- #27 
- #30 

## Possible Solution(s)
Sync changes from 'main' branch to prevent

### Objectives
- [x] Create a new branch for the issue
- [x] Resolve any conflicts with existing changes
- [x] Remove any references to old information about:
  - [x] Development cycle
  - [x] Boards
  - [x] Scrum
  - [x] Kanban
- [x] Include changes from #39
  After successfully merge of branch 'bug-39-FixBrokenLink', branch 'wip-38-SyncSeparationOfConcernsWithMain' needs to sync changes from 'main'
- [x] Submit a Pull Request to merge into 'wip-7-SeparationOfConcerns', only when:
  - [x] Branch 'bug-39-FixBrokenLink' has merged 
  - [x] #39 has been resolved

**Comment by fjbo-net** [comment]
Blocks:
- #30

**Comment by fjbo-net** [comment]
Blocked by:
- #39
  After successfully merge of branch, branch 'wip-38-SyncSeparationOfConcernsWithMain' needs to sync changes from 'main'

**Comment by fjbo-net** [comment]
Blocks:
- #39

**Comment by fjbo-net** [comment]
No longer blocked by:
- #39

**Comment by fjbo-net** [comment]
No longer blocks:
- #39
